Autopilot/Utilities.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Data.SqlClient;
+using System.Windows;
+
+namespace Autopilot
+{
+    class Utilities
+    {
+        public string DBconnStrg = Properties.Settings.Default.AutopilotConnectionString; //zentraler DB-ConnectionStrg
+
+        public void SQLCmd(string SQLCmd_txt)
+        {
+            SqlConnection conn = new SqlConnection(DBconnStrg);
```

Comment:
Die Sache mit dem SQL brauchen wir nur für UPDATE, DELETE, INSERT..., wenn Daten durch einen Button geändert werden sollen (z.B. neuer Auftrag = INSERT in Datenbank). Hatte ich jetzt so gedacht.
Die ganz normalen SELECT-Anweisungen (füllen von DataGrids u.ä.) lassen sich natürlich schöner über das Framework machen.